pkg/log/logger.go

```diff
@@ -23,6 +23,9 @@ const (
 	PrefixMisconfiguration = "misconfig"
 	PrefixSecret           = "secret"
 	PrefixLicense          = "license"
+	PrefixVulnerabilityDB  = "vulndb"
+	PrefixJavaDB           = "javadb"
+	PrefixChecksBundle     = "checks"
```

Comment:
`checks` looks too generic.
Perhaps we want to use `checks-bundle` for this?

Reply:
Good point, I was actually thinking about it and didn't have a good idea. Then, I used `checks`. On second thoughts, I chose `misconfig`.
https://github.com/aquasecurity/trivy/pull/7625/commits/d8bfa99829c98458d12385f9215efb52eb0f688a